lms/djangoapps/teams/static/teams/js/spec/views/topic_teams_spec.js

```diff
@@ -65,17 +62,16 @@ define([
             var emptyMembership = TeamSpecHelpers.createMockTeamMemberships([]),
                 teamsView = createTopicTeamsView({ teamMemberships: emptyMembership });
             spyOn(Backbone.history, 'navigate');
-            teamsView.$('a.browse-teams').click();
+            teamsView.$('.browse-teams').click();
             expect(Backbone.history.navigate.calls[0].args).toContain('browse');
         });
 
-        it('can search teams', function () {
+        it('gives the search field focus when clicking on the search teams link', function () {
             var emptyMembership = TeamSpecHelpers.createMockTeamMemberships([]),
                 teamsView = createTopicTeamsView({ teamMemberships: emptyMembership });
-            spyOn(Backbone.history, 'navigate');
-            teamsView.$('a.search-teams').click();
-            // TODO! Should be updated once team description search feature is available
-            expect(Backbone.history.navigate.calls[0].args).toContain('browse');
+            spyOn($.fn, 'focus').andCallThrough();
+            teamsView.$('.search-teams').click();
```

Comment:
Should it do anything with text that is already in the search box? The caret position seems odd now.